docfx_yaml/monkeypatch.py

```diff
@@ -6,34 +6,35 @@
 from sphinx.util import docfields
 from sphinx import directives, addnodes
 from sphinx import addnodes
-
 from sphinx.addnodes import desc, desc_signature
 from .utils import transform_node as _transform_node
 from .nodes import remarks
+from sphinx.util.logging import getLogger 
+
+logger = getLogger("sphinx-docfx-yaml")
+
 
 TYPE_SEP_PATTERN = '(\[|\]|, |\(|\))'
 REF_PATTERN = ':(func|class|meth|mod|ref|any):`~?([a-zA-Z_\.<> ]*?)`'
 
 def _get_desc_data(node):
     assert node.tagname == 'desc'
     if node.attributes['domain'] != 'py':
-        print(
-            'Skipping Domain Object (%s)' % node.attributes['domain']
-        )
+        logger.info("[docfx] Skipping Domain Object (%s)' % node.attributes['domain']")
         return None, None
 
     try:
         module = node[0].attributes['module']
         full_name = node[0].attributes['fullname'].split('.')[-1]
     except KeyError as e:
-        print("[docfx_yaml] There maybe some syntax error in docstring near: " + node.astext())
+        logger.warning("[docfx_yaml] There maybe some syntax error in docstring near: '{}'".format(node.astext()))
```

Comment:
Add "warning" to warn message

Reply:
The warning is usually added by the logging system.